cfgov/paying_for_college/disclosures/urls.py

```diff
@@ -25,8 +27,12 @@
                 template_name="paying_for_college/disclosure_technote.html"),
             name="pfc-technote"),
 
-    re_path(r"^api/search-schools.json", school_search_api,
-            name="school_search"),
+    flagged_re_path(
+        'ELASTICSEARCH_DSL_PFC',
+        r"^api/search-schools.json",
+        school_search,
```

Comment:
The app only uses an autocomplete search box, so we need to point the url at the `school_autocomplete` view:

```suggestion
        school_autocomplete,
```

Reply:
Done. I changed `school_search` to `school_autocomplete` in `urls.py`.